.gitignore

```diff
@@ -20,6 +20,3 @@ bin
 vendor/
 
 /config/
-
-# only for kubebuilder
-main.go
```

Comment:
fixed

Reply:
why add a `/`?

Reply:
Because if it's not a path, all the `main.go` will be ignored, e.g., `cmd/daemon/main.go`. After I rename `cmd/daemon/daemon.go` to `cmd/daemon/main.go`, I found that it cannot be added to commit.